cl/scrapers/management/commands/scrape_caselaw.py

```diff
@@ -0,0 +1,339 @@
+            "argument to True. Exiting. "
+        )
+        return
+
+    if source and source not in ["Harvard", "Fastcase"]:
+        logger.error(
+            "Invalid source. Valid sources: Harvard or Fastcase. Exiting. "
+        )
+        return
+
+    if type == "jurisdictions":
+        endpoint = f"{base_url}/{version}/jurisdictions/"
+    elif type == "courts":
+        endpoint = f"{base_url}/{version}/courts/"
+    elif type == "reporters":
+        endpoint = f"{base_url}/{version}/reporters/"
+    elif type == "cases":
+        endpoint = f"{base_url}/{version}/cases/"
+        params["page_size"] = page_size
+        if ordering:
+            params["ordering"] = ordering
+        if court:
+            params["court"] = court
+        if jurisdiction:
+            params["jurisdiction"] = jurisdiction
+        if full_case:
+            params["full_case"] = "true" if full_case else "false"
+        if body_format:
+            params["body_format"] = body_format
+        if last_updated and last_updated_filter:
+            params[f"last_updated__{last_updated_filter}"] = last_updated
+        if reporter_id:
+            params["reporter"] = reporter_id
+        if source:
+            params["source"] = source
+
+    # Directory to store data based on type
+    directory = Path(settings.MEDIA_ROOT, "case.law", type)
+
+    if cursor:
+        # Start from specific cursor
+        params["cursor"] = cursor
+
+    while endpoint:
+        data = get_data(endpoint, params=params)
+        # We only need to pass the params on first request
+        params = {}
+
+        if data:
+            results_count = data.get("count", None)
+            old_endpoint = endpoint
+            endpoint = data.get("next", None)
+            results = data.get("results", [])
+
+            for result in results:
+                # We go through each result
+                id = result.get("id")
+                filename = f"{id}.json"
+                if type == "cases":
+                    volume_number = (
+                        result.get("volume").get("volume_number")
+                        if result.get("volume")
+                        else None
+                    )
+                    first_page = result.get("first_page")
+                    reporter_id = (
+                        result.get("reporter").get("id")
+                        if result.get("reporter")
+                        else None
+                    )
+                    reporter_url = (
+                        result.get("reporter").get("url")
+                        if result.get("reporter")
+                        else None
+                    )
+
+                    reporter_slug = get_reporter_slug(reporter_url)
+
+                    if not reporter_slug:
+                        # No slug from url, use reporter id
+                        reporter_slug = reporter_id
```

Comment:
I don't understand this.  Are there examples for cases that don't have reporters?